packages/shared_preferences/shared_preferences/example/lib/main.dart

```diff
@@ -61,14 +65,25 @@ class SharedPreferencesDemoState extends State<SharedPreferencesDemo> {
     });
   }
 
+  Future<void> _migratePreferences() async {
+    // #docregion migrate
+    const SharedPreferencesOptions sharedPreferencesOptions =
+        SharedPreferencesOptions();
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
+    await migrateLegacySharedPreferencesToSharedPreferencesAsyncIfNecessary(
+        prefs, sharedPreferencesOptions, 'migrationCompleted');
```

Comment:
Since this is a README snippet, where there's no context about the arguments, let's make a
`const String migrationCompletedPrefKey = 'migrationCompleted'` as the previous line, and then use that variable here, so it's self-documenting in the README.